### PR TITLE
add capacity attribute to event_slot_serializer

### PR DESCRIPTION
**Pull request by maheshkurma**
added event_slot_controller.rb (show action)
add capacity attribute to event_slot_serializer and event_hour_serializer

### CODE REVIEW

File: app/controllers/api/event_slots_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Api
+  # Controller to expose Event Dates
+  class EventSlotsController < Api::BaseController
+    before_action :set_event_slot, only: [:show]
+
+    # GET /api/event_dates/:id
```
Comment (PhilNorman2) [inline app/controllers/api/event_slots_controller.rb:8]: Should be # GET /api/event_slots/:id

Comment (maheshkurma) [reply app/controllers/api/event_slots_controller.rb]: updated @PhilNorman2 thanks !

File: spec/controllers/api/event_hours_controller_spec.rb [hunk 2]
```diff
@@ -53,6 +53,7 @@
   def expected_event_slots
     {
       'event_slot_id' => 1,
+      'capacity' => 5,
       'start_time' => '10 AM',
       'end_time' => '11 AM',
       'open_slots' => 5
```
Comment (PhilNorman2) [inline spec/controllers/api/event_hours_controller_spec.rb:59]: Suggestion: Consider adding reserved value to expected_event_slots, e.g., through factory and show open_slots expected results as capacity - reserved.

File: spec/controllers/api/event_hours_controller_spec.rb [hunk 3]
```diff
@@ -74,11 +75,13 @@ def expected_response
         'date' => date,
         'event_hours' => [{
           'event_hour_id' => event_hour.event_hour_id,
+          'capacity' => 5,
```
Comment (PhilNorman2) [inline spec/controllers/api/event_hours_controller_spec.rb:78]: Suggestion: Consider adding reserved value to expected_event_slots, e.g., through factory and show open_slots expected results as capacity - reserved.

**Pull request merged**